exporters/holstebro.py

```diff
@@ -0,0 +1,123 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+"""
+Holstebro Kommune specific queries into MO.
+"""
+import json
+import logging
+import os
+import pathlib
+import queue
+import threading
+import time
+
+from datetime import datetime, timedelta
+
+import click
+import requests
+from anytree import Node, PreOrderIter
+
+import common_queries as cq
+import holstebro_helpers as hh
+import holstebro_logger
+from os2mo_helpers.mora_helpers import MoraHelper
+
+cfg_file = pathlib.Path.cwd() / 'settings' / 'holstebro.settings.json'
+if not cfg_file.is_file():
+    raise Exception('No setting file')
+SETTINGS = json.loads(cfg_file.read_text())
+
+logger = logging.getLogger('Exports')
+
+
+@click.command()
+@click.option('--dev', 'hostname', flag_value='https://os2mo-dev.holstebro.dk', help='https://os2mo-dev.holstebro.dk')
+@click.option('--test', 'hostname', flag_value='https://os2mo-test.holstebro.dk', help='https://os2mo-test.holstebro.dk')
+@click.option('--prod', 'hostname', flag_value='https://os2mo.holstebro.dk', help='https://os2mo.holstebro.dk')
+def export_from_mo(hostname):
+    threaded_speedup = False
+    t = time.time()
+    holstebro_logger.start_logging(SETTINGS['logging.holstebro.exporters_logfile'])
+
+    mh = MoraHelper(hostname=hostname, export_ansi=False)
+
+    logger.info(f"Reading organisation from: {hostname}")
+    org = mh.read_organisation()
+
+    # find Holstebro Kommune root uuid, if no uuid is specified
+    org_roots = mh.read_top_units(org)
+    root_uuids = []
+    for root in org_roots:
+        root_uuids.append(root['uuid'])
```

Comment:
Der her ligner at `root_uuids` blot indeholder `uuid`s for alle elementer i `org_roots`.
Jeg ville nok skrive:
```
    root_uuids = list(map(itemgetter('uuid'), org_roots))
```
Eller via en list comprehension:
```suggestion
    root_uuids = [root['uuid'] for root in org_roots]
```